source/engine/graphics/Graphics.cpp

```diff
@@ -1,42 +1,48 @@
 #include "Graphics.h"
+#include "FrameManager.h"
+#include "GpuAllocator.h"
+#include "GpuAssetsStorage.h"
+#include "GpuOptions.h"
+#include "Imgui.h"
+#include "Lighting.h"
+#include "RenderGraph.h"
+#include "Swapchain.h"
+#include "core/Device.h"
+#include "core/Instance.h"
+#include "shaders/ShaderDescriptorSets.h"
+#include "shaders/ShaderResources.h"
 #include "config/Config.h"
 #include "ecs/Registry.h"
 #include "graphics/Camera.h"
```

Comment:
missed this one (graphics/Camera)

Reply:
That one is actually not a sibling file. Its include/graphics/Camera.h.